docs/source/api_reference/divicon.rst

```diff
@@ -0,0 +1,33 @@
+DivIcon
+======
+
+Example
+-------
+
+.. jupyter-execute::
+
+    from ipyleaflet import Marker, DivIcon, Map
+
+    center = (52.204793, 360.121558)
+
+    m = Map(center=center, zoom=10)
+    icon = DivIcon(html='foo bar', bg_pos=[0, 0], icon_size=[150, 150])
+    mark = Marker(location=center, icon=icon)
+    m.add_layer(mark);
+
+    m
+
+
+Attributes
+----------
+
+=====================    =====================   ===
+Attribute                Default Value           Doc
+=====================    =====================   ===
+html                     ''                      Custom HTML code to put inside the div element,
+bg_pos                   (0, 0)                  Optional relative position of the background, in pixels.
+icon_size                (10, 10)                size icon will be rendered
+icon_anchor              (0, 0)                  anchor point of icon
+class_name               leaflet-div-icon        A custom class name to assign to icon.
```

Comment:
I know this is here in some classes already, but I actually think we should not expose any `class_name` property to Python.

The reason is that users don't have a non-hacky way to inject CSS into the page, so I think we should not rely on this property.

Reply:
Yes right makes sense.